Dependencies/AndroidExtensions/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(SOURCES
+    "Include/AndroidExtensions/Globals.h"
+    "Include/AndroidExtensions/JavaWrappers.h"
+    "Source/Globals.cpp"
+    "Source/JavaWrappers.cpp")
+
+add_library(AndroidExtensions ${SOURCES})
+
+target_include_directories(AndroidExtensions PRIVATE "Include")
+target_include_directories(AndroidExtensions INTERFACE "Include")
```

Comment:
nit: If both of these are the same, can you just make one PUBLIC one?

Reply:
oh, good point.